Extensions/TileMapObject/TileMapPanel.cpp

```diff
@@ -139,7 +143,16 @@ void TileMapPanel::OnPaint(wxPaintEvent& event)
                 if(m_tilemap->GetTile(layer, col, row) == -1)
                     continue;
 
-                dc.DrawBitmap(m_tileset->GetTileBitmap(m_tilemap->GetTile(layer, col, row)), GetPositionOfTile(col, row).x, GetPositionOfTile(col, row).y, true);
+                dc.Blit(GetPositionOfTile(col, row).x, 
+                        GetPositionOfTile(col, row).y,
+                        m_tileset->tileSize.x,
+                        m_tileset->tileSize.y,
+                        &tilesetDC,
+                        m_tileset->GetTileTextureCoords(m_tilemap->GetTile(layer, col, row)).topLeft.x,
+                        m_tileset->GetTileTextureCoords(m_tilemap->GetTile(layer, col, row)).topLeft.y,
+                        wxCOPY,
+                        true);
+                //dc.DrawBitmap(m_tileset->GetTileBitmap(m_tilemap->GetTile(layer, col, row)), GetPositionOfTile(col, row).x, GetPositionOfTile(col, row).y, true);
```

Comment:
You should delete this line :) If Blit works properly, there is not reason to keep this line and if we need to go back to the previous method for whatever reason we can just inspect the git ;)